src/Type/Definition/AbstractTypeInterface.php

```diff
@@ -2,7 +2,10 @@
 
 namespace Digia\GraphQL\Type\Definition;
 
-interface AbstractTypeInterface extends NamedTypeInterface
+use GraphQL\Contracts\TypeSystem\Type\NamedTypeInterface;
+use GraphQL\Contracts\TypeSystem\Type\AbstractTypeInterface as AbstractTypeContract;
```

Comment:
Why change the name?

Reply:
Do you mean `AbstractTypeContract`?

Reply:
Yes

Reply:
There is `interface AbstractTypeInterface extends AbstractTypeContract {}`